zenodo/modules/records/templates/zenodo_records/record_detail.html

```diff
@@ -27,19 +27,22 @@
 {%- from "zenodo_records/macros/accessright.html" import accessright_label -%}
 {%- from "zenodo_records/macros/objecttype.html" import objecttype_label -%}
 {%- from "zenodo_records/macros/altmetric.html" import almetric_badge -%}
-{%- from "zenodo_records/macros/authors.html" import authors -%}
+{%- from "zenodo_records/macros/authors.html" import authors as authors_macro -%}
```

Comment:
Otherwise the macro name is colliding with the variable set here https://github.com/JavierDelgadoFernandez/zenodo/blob/3d0f2cd6b64de3a0b05ea23f385eab0ef4f5552e/zenodo/modules/records/templates/zenodo_records/record_detail.html#L38